test/unit/util.js

```diff
@@ -1243,4 +1243,17 @@
     ];
     assert.deepEqual(matrix, expected, 'dimensions matrix flipped is equal');
   });
+
+  QUnit.test.only('getElementOffset inside iframe', function (assert) {
+    var frame = fabric.document.createElement('iframe');
+    fabric.document.body.appendChild(frame);
+    var iframeDoc = frame.contentDocument || frame.contentWindow.document;
+    var el = fabric.document.createElement('canvas');
+    iframeDoc.body.appendChild(el);
+    frame.style.left = '200px';
+    el.style.left = '200px';
+    assert.equal(fabric.util.getElementOffset(el).left, 400);
```

Comment:
@asturur @YashasviChaurasia I don't understand what the outcome should be...
See failing tests